app/(public)/components/paragraph/sul-contact-card/node-reference-card.tsx

```diff
@@ -79,64 +74,50 @@ const toISOStringWithTimezone = date => {
               <h2 className="su-type-3 su-rs-mb-1 su-text-white">{paragraph.sul_contact__title}</h2>
             )}
             <div className="su-leading-tight md:su-rs-px-2 su-text-white">
-              <Conditional showWhen={paragraph.sul_contact__hours || paragraph.sul_contact__branch?.su_library__hours}>
+              <Conditional showWhen={paragraph.sul_contact__branch?.su_library__hours}>
                 <div className="su-relative su-flex su-flex-row su-items-start su-rs-mb-0 su-type-1">
                   <ClockIcon width={19} className="md:su-absolute md:su-left-[-38px] md:su-top-01em su-mr-12 su-flex-shrink-0"/>
-                  <div className="sm:su-flex">
-                    <Conditional showWhen={paragraph.sul_contact__hours}>
-                      <div className="su-text-white su-rs-mb-neg2 sm:su-mb-0">
-                        {paragraph.sul_contact__hours}								
-                      </div>
-                    </Conditional>
-                    {paragraph.sul_contact__link?.url && 
-                      <Link className={"su-underline su-text-white hocus:su-text-illuminating-dark hocus:su-no-underline active:su-text-digital-red-light su-font-normal" + (paragraph.sul_contact__hours && " sm:su-ml-15") } href={paragraph.sul_contact__link?.url}>
-                        {paragraph.sul_contact__link?.title}
-                      </Link>
-                    }
-                  </div>
-                  <Conditional showWhen={paragraph.sul_contact__branch?.su_library__hours}>
-                    <div className="su-text-white" aria-live="polite">
-                      <div className="su-flex">
-                        {isOpen ? 'Open ' : 'Closed '}
-                        /
-                        <div className="su-ml-5">
-                          {!closedAllDay && (isOpen ? 'Closes at ' + closeTime?.toLocaleTimeString("en-US", {
-                            hour: "numeric",
-                            minute: "numeric",
-                          }) : 'Opens at ' + openTime?.toLocaleTimeString("en-US", {
-                            hour: "numeric",
-                            minute: "numeric",
-                          }))}
-                        </div>
+                  <div className="su-text-white" aria-live="polite">
+                    <div className="su-flex">
+                      {isOpen ? 'Open ' : 'Closed '}
+                      /
+                      <div className="su-ml-5">
+                        {!closedAllDay && (isOpen ? 'Closes at ' + closeTime?.toLocaleTimeString("en-US", {
+                          hour: "numeric",
+                          minute: "numeric",
+                        }) : 'Opens at ' + openTime?.toLocaleTimeString("en-US", {
+                          hour: "numeric",
+                          minute: "numeric",
+                        }))}
```

Comment:
Let's make sure to add the `timezone` into these. see it in the [options](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString#using_options)

Reply:
Also sometimes it's more clear to put this logic above so that you just print variables in the markup.